chalice/app.py

```diff
@@ -227,19 +236,19 @@ def __repr__(self):
 
 
 class Authorizer(object):
-    name = ''
-    scopes = []
+    name: str = ''
+    scopes: List[str] = []
 
-    def to_swagger(self):
+    def to_swagger(self) -> Dict[str, Any]:
         raise NotImplementedError("to_swagger")
 
-    def with_scopes(self, scopes):
+    def with_scopes(self, scopes: List[str]) -> 'Authorizer':
```

Comment:
Since `from __future__ import annotations` is only supported for python 3.7+ 
To support v3.6, We are using quotes for annotations referring to classes declared after usage. Example - `'Authorizer'`